### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aaditgupta21**
Just add ./aeacus configure to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 1. __Set up the environment.__
     - Put your __config__ in `/opt/aeacus/scoring.conf` or`C:\aeacus\scoring.conf`.
+    - Run `./aeacus configure` to configure Vulns, Only for Linux :(
```
Comment (safinsingh) [inline README.md]: (v)ulns, (o)nly and maybe remove the `:(`

Comment (safinsingh) [reply README.md]: - Run `./aeacus configure` to launch GUI tool for configuring vulnerabilities

Comment (aaditgupta21) [reply README.md]: Done

**Review by safinsingh — DISMISSED** [review]
See comment

**Review by aaditgupta21 — COMMENTED** [review]
Done

**Review by safinsingh — APPROVED** [review]

**Pull request merged**